### PR TITLE
GHC-58008

### PR DESCRIPTION
**Pull request by Soupstraw**

### CODE REVIEW

File: message-index/messages/GHC-58008/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Pattern matching on GADTs without MonoLocalBinds is fragile
+summary: Pattern matching on GADTs without MonoLocalBinds is fragile
+introduced: 9.6.1
+severity: warning
+flag: -Wgadt-mono-local-binds
+---
+
+This warning is triggered on pattern matching involving GADTs, if `MonoLocalBinds` is disabled. Type inference can be fragile in this case.
+See the OutsideIn(X) paper and Let-generalisation for more details.
```
Comment (david-christiansen) [inline message-index/messages/GHC-58008/index.md:10]:
```suggestion
See [the OutsideIn(X) paper](https://doi.org/10.1017/S0956796811000098) and Let-generalisation for more details.
```

Also, what is "Let-generalisation" referring to here? Is it a section of the paper, a section of the user's guide, or something else?

**Review by david-christiansen — CHANGES_REQUESTED** [review]
Thank you!

I suggest adding a link to the paper you're citing. The other documentation reference didn't make sense to me - can you clarify?

File: message-index/messages/GHC-58008/index.md (hunk 1 above)
Comment (Soupstraw) [reply message-index/messages/GHC-58008/index.md]: Fixed the links

**Review by david-christiansen — APPROVED** [review]

**Comment by david-christiansen** [comment]
Thanks!

**Pull request merged**